component-samples/map-components/samples/angular/src/app/app.component.ts

```diff
@@ -13,13 +13,24 @@
  * limitations under the License.
  */
 
-import { Component, OnInit } from "@angular/core";
-import { defineCustomElements } from "@arcgis/map-components/dist/loader";
+import { Component, CUSTOM_ELEMENTS_SCHEMA, OnInit } from "@angular/core";
+import { RouterOutlet } from "@angular/router";
+
+import "@arcgis/map-components/dist/components/arcgis-map";
+import "@arcgis/map-components/dist/components/arcgis-zoom";
+import "@arcgis/map-components/dist/components/arcgis-search";
+import "@arcgis/map-components/dist/components/arcgis-legend";
+import { setAssetPath as setCalciteComponentsAssetPath } from '@esri/calcite-components/dist/components';
+
+setCalciteComponentsAssetPath("https://js.arcgis.com/calcite-components/2.8.6/assets");
```

Comment:
@manugadde move setting the asset path into the constructor, so that its set at the same time the class is instantiated. 

For example:

```ts
constructor() {
   setCalciteComponentsAssetPath("https://js.arcgis.com/calcite-components/2.8.6/assets");
 }